addons/account/views/account_move_views.xml

```diff
@@ -583,8 +583,14 @@
                         <button name="action_invoice_sent"
                                 type="object"
                                 string="Send &amp; Print"
-                                attrs="{'invisible':['|', '|', ('state', '!=', 'posted'), ('invoice_pdf_report_id', '!=', False), ('move_type', 'not in', ('out_invoice', 'out_refund'))]}"
-                                class="oe_highlight"
+                                attrs="{'invisible':['|', '|', '|', ('is_being_sent', '=', True), ('state', '!=', 'posted'), ('invoice_pdf_report_id', '!=', False), ('move_type', 'not in', ('out_invoice', 'out_refund'))]}"
+                                class="btn btn-primary"
+                                data-hotkey="y"/>
+                        <button name="action_invoice_sent"
+                                type="object"
+                                string="Send &amp; Print"
+                                attrs="{'invisible':['|', '|', '|', ('is_being_sent', '=', False), ('state', '!=', 'posted'), ('invoice_pdf_report_id', '!=', False), ('move_type', 'not in', ('out_invoice', 'out_refund'))]}"
+                                class="btn btn-secondary"
```

Comment:
couldn't you just add the condition is_being_sent=False on the next button?


and the class? why is there no class on the next button? what's the default? :thinking:

Reply:
Good point, the default class of `<button>` is apparently `btn btn-secondary`.